__tests__/ClusterManagement.js

```diff
@@ -0,0 +1,204 @@
+import '@testing-library/jest-dom/extend-expect';
+import {
+  API_ENDPOINT,
+  appCatalogsResponse,
+  getPersistedMockCall,
+  infoResponse,
+  ORGANIZATION,
+  orgResponse,
+  orgsResponse,
+  releasesResponse,
+  userResponse,
+  v5ClustersResponse,
+  v4AWSClusterStatusResponse,
+  appsResponse,
+  V4_CLUSTER,
+  v4ClustersResponse,
+  v4AWSClusterResponse,
+  v5ClusterResponse,
+  nodePoolsResponse,
+} from 'test_utils/mockHttpCalls';
+import { fireEvent, wait } from '@testing-library/react';
+import { renderRouteWithStore } from 'test_utils/renderUtils';
+import nock from 'nock';
+import { within } from '@testing-library/dom';
+
+// Tests setup
+const requests = {};
+
+// Responses to requests
+beforeAll(() => {
+  requests.userInfo = getPersistedMockCall('/v4/user/', userResponse);
+  requests.releases = getPersistedMockCall('/v4/releases/', releasesResponse);
+  requests.info = getPersistedMockCall('/v4/info/', infoResponse);
+  requests.organizations = getPersistedMockCall(
+    '/v4/organizations/',
+    orgsResponse
+  );
+  requests.organization = getPersistedMockCall(
+    `/v4/organizations/${ORGANIZATION}/`,
+    orgResponse
+  );
+  requests.clusters = getPersistedMockCall('/v4/clusters/');
+  requests.credentials = getPersistedMockCall(
+    `/v4/organizations/${ORGANIZATION}/credentials/`
+  );
+  requests.appcatalogs = getPersistedMockCall(
+    '/v4/appcatalogs/',
+    appCatalogsResponse
+  );
+});
+
+// Stop persisting responses
+afterAll(() => {
+  Object.keys(requests).forEach(req => {
+    requests[req].persist(false);
+  });
+});
+
+/************ TESTS ************/
+it('creates a v5 cluster and redirect to details view', async () => {
+  const v5ClusterCreationResponse = {
+    code: 'RESOURCE_CREATED',
+    message: `The cluster with ID ${v5ClusterResponse.id} has been created.`,
+  };
+
+  // Cluster POST request
+  const v5ClusterCreationRequest = nock(API_ENDPOINT)
+    .intercept(`/v5/clusters/`, 'POST')
+    .reply(200, v5ClusterCreationResponse, {
+      location: `/v5/clusters/${v5ClusterResponse.id}/`,
+    });
+
+  // Node Pools POST response
+  const nodePoolCreationResponse = { ...nodePoolsResponse[0] };
+
+  // Node pools POST request
+  const nodePoolCreationRequest = nock(API_ENDPOINT)
+    .intercept(`/v5/clusters/${v5ClusterResponse.id}/nodepools/`, 'POST')
+    .reply(200, nodePoolCreationResponse);
+
+  // Node pools get
+  const nodePoolsRequest = getPersistedMockCall(
+    `/v5/clusters/${v5ClusterResponse.id}/nodepools/`,
+    [nodePoolCreationResponse]
+  );
+
+  // Clusters GET request
+  requests.clusters = getPersistedMockCall('/v4/clusters/', [
+    v5ClusterResponse,
+  ]);
+
+  // Cluster GET request
+  const clusterRequest = getPersistedMockCall(
+    `/v5/clusters/${v5ClusterResponse.id}/`,
+    v5ClusterResponse
+  );
+
+  const div = document.createElement('div');
```

Comment:
As we talked on the previous PR, this is not actually needed, since Testing Library creates its own element to render in

Reply:
:+1: agree,  I was used to add the container before when using `ReactDOm` for tests, but this is not needed here, let's get rid of this.

Reply:
But I suggest it would be best to merge this first, and then I will add a PR modifying the function and all the calls.